openslides/core/static/js/core/base.js

```diff
@@ -39,40 +39,41 @@ angular.module('OpenSlidesApp.core', [
     'DS',
     '$rootScope',
     function (DS, $rootScope) {
-        var url = location.origin + "/sockjs";
-        var socket;
+        var socket = null;
+        var recInterval = null;
+        $rootScope.connected = true;
+
+        var newConnect = function () {
+            socket = new SockJS(location.origin + "/sockjs");
+            clearInterval(recInterval);
+            socket.onopen = function () {
+                $rootScope.connected = true;
+            };
+            socket.onclose = function () {
+                $rootScope.connected = false;
```

Comment:
How can I prevent that $rootScope.connected is set to false after reload of the page? I don't want to show the red bar on reload - only for connection failed during event.